pkg/test/framework/components/ingress/ingress.go

```diff
@@ -30,12 +39,16 @@ type Instance interface {
 	Address() string
 
 	//  Call makes an HTTP call through ingress, where the URL has the given path.
-	Call(path string) (CallResponse, error)
-	CallOrFail(t test.Failer, path string) CallResponse
+	Call(path, host string) (CallResponse, error)
+	CallOrFail(t test.Failer, path, host string) CallResponse
 }
 
 type Config struct {
-	Istio istio.Instance
+	Istio       istio.Instance
+	IngressType IgType
+	CaCert      string
```

Comment:
Need some documentation about these are file path, secret name or inlined base64 bytes.

Reply:
Even better, if it's one of them, consider to use some existing types.

Reply:
Added comments describing these fields.